app/threads/client/flextab/thread.js

```diff
@@ -131,15 +125,37 @@ Template.thread.onRendered(function() {
 		this.chatMessages.initializeInput(this.find('.js-input-message'), { rid, tmid });
 	});
 
-	Tracker.afterFlush(() => {
-		this.autorun(async () => {
-			const { mainMessage } = Template.currentData();
-			this.state.set({
-				tmid: mainMessage._id,
-				rid: mainMessage.rid,
-			});
+
+	this.autorun(async () => {
+		const { mainMessage, jump } = Template.currentData();
+		this.state.set({
+			tmid: mainMessage._id,
+			rid: mainMessage.rid,
+			jump,
 		});
 	});
+
+	this.autorun(() => {
+		const jump = this.state.get('jump');
+		const loading = this.state.get('loading');
+
+		if (jump && loading === false) {
+			this.find('.js-scroll-thread').style.scrollBehavior = 'smooth';
+			this.state.set('jump', null);
```

Comment:
```suggestion
			this.state.set('jump', null);
```

Accidental emoji?

Reply:
mac os loves to add this invisible char after `''`